CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(tree-sitter-nuon
+        VERSION "0.21.0"
```

Comment:
@melMass should the version be something like 0.2.0?

Reply:
Do you mean as a bump? Maybe 0.22.0 then yep!